sdcm/nemesis.py

```diff
@@ -1210,22 +1210,10 @@ def disrupt_modify_table(self):
 
     def disrupt_mgmt_backup(self):
         self._set_current_disruption('ManagementBackup')
-        if not self.cluster.params.get('use_mgmt', default=None):
```

Comment:
i think this `if` should be re-added, as this nemesis is useless if `use_mgmt` is not `True`, and then there is no need to go further and call the new placed function `self.cluster.get_cluster_manager()`.

Reply:
I agree with @fgelcer, this code was raising `UnsupportedNemesis` , the new code would raise `ScyllaManagerError`.
this would fail any job that would run this

Reply:
if a test configures use_mgmt=false and nemesis = MgmtRepair then it sounds fair enough for the nemesis to fail with ScyllaManagerError. it may also be possible to raise 'unsupported-nemesis' somehow as well.
secondly, it cannot just be re-added alone since it won't work for cloud, of course. this has a wider scope since use_mgmt cannot apply currently for cloud manager (cloud manager is installed by a different env var). if we want it to apply 'use_mgmt', then, for example, some additional configuration should be added to the pipelines to utilize it. 
so we can open a discussion whether this parameter should apply to cloud manager as well or not.

Reply:
this isn't correct, as nemesis are called randomly (in most cases), and in some specific configurations we don't want to run with `use_mgmt=true`, and the nemesis could still be called.
It is possible to call `mgr_cluster = self.cluster.get_cluster_manager()` using `try .. except` and raise the `unsupported nemesis` when it gets to `ScyllaManagerError`.
but the nemesis can't fail with `ScyllaManagerError` since the beginning, as the test didn't even set to `use_mgmt=true`

Reply:
@yarongilor let's not change the current semantics of the nemesis, we could just move things around, but without changing the current logic.

As for the configuration of the cloud that should be figured out on the sct_plugin end

Reply:
ok, fixed by adding new sct param of "use_cloud_manager"

Reply:
why did you need to add a new sct param to fix it?
as you mentioned before, the cloud clusters don't use manager, so this will be false, hence this nemesis will raise an `unsupportedNemesis` exception and everything will continue on

Reply:
it won't be false since nemesis changed to check:
```
if not self.cluster.params.get('use_mgmt', default=None) and not self.cluster.params.get('use_cloud_manager',
                                                                                                 default=None)
```